<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="dev" name="Command line test" tests="1" failures="1" timestamp="11 Feb 2014 07:23:28 GMT" time="0.02" errors="0">
  <testcase name="add" time="0.002" classname="com.kabaso.cput.tp.CalculatorTest">
    <failure type="junit.framework.AssertionFailedError" message="Add 2+3 expected:&amp;lt;5&amp;gt; but was:&amp;lt;6&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: Add 2+3 expected:<5> but was:<6>
	at junit.framework.Assert.fail(Assert.java:50)
	at junit.framework.Assert.failNotEquals(Assert.java:287)
	at junit.framework.Assert.assertEquals(Assert.java:67)
	at junit.framework.Assert.assertEquals(Assert.java:199)
	at com.kabaso.cput.tp.CalculatorTest.add(CalculatorTest.java:33)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- add -->
</testsuite> <!-- Command line test -->
